src/Streamly/Internal/Data/Fold.hs

```diff
@@ -19,11 +19,11 @@ module Streamly.Internal.Data.Fold
     , Fold (..)
 
     -- * Constructors
-    , mkFoldl
-    , mkFoldlM
-    , mkFoldl1
-    , mkFoldr
-    , mkFoldrM
+    , foldl'
+    , foldlM'
+    , foldl1
```

Comment:
I am sorry, this had to be `foldl1'` instead. The same as we have in "Streamly.Prelude".